model/distro/distro.go

```diff
@@ -649,14 +654,16 @@ func (d *Distro) GetResolvedHostAllocatorSettings(s *evergreen.Settings) (HostAl
 	config := s.Scheduler
 	has := d.HostAllocatorSettings
 	resolved := HostAllocatorSettings{
-		Version:                has.Version,
-		MinimumHosts:           has.MinimumHosts,
-		MaximumHosts:           has.MaximumHosts,
-		AcceptableHostIdleTime: has.AcceptableHostIdleTime,
```

Comment:
This actually makes it look like the ticket description "AcceptableHostIdleTime is unused" is untrue.  Do we use this anywhere?

Reply:
This was also unused prior to this PR

Reply:
Resolving this thread bc I don't remember what I meant haha